### PR TITLE
Couple more i18n adjustments

### PR DESCRIPTION
**Pull request by svetlemodry**
Fixing a typo (or two :)
And back porting some of the ammendments I did for my web to improve the i18n of the web page.

### CODE REVIEW

File: i18n/en.yaml [hunk 1]
```diff
@@ -24,3 +24,12 @@
...
     other: "{{ .Count }} posts"
+
+- id: sidebarCategories
+  translation: "Categories"
+
+- id: sidebarFeaturedTags
+  translation: "Featured Tags"
+
+- id: sidebarRecommendedSeries
+  translation: "Recommend Series"
```
Comment (EliiseS) [inline i18n/en.yaml]: This looks like a typo too, should be `RecommendedSeries` indeed. Just like the the variable name

**Comment by EliiseS** [comment]
Your commit messages are really nice, could you add a description for the PR too? 😄

File: i18n/en.yaml (hunk 1 above)
Comment (svetlemodry) [reply i18n/en.yaml]: team work, good catch!

**Comment by EliiseS** [comment]
You've also got merge conflicts now, due to your other PR merging. As soon as you can fix those I'll 🚢 @svetlemodry

**Review by EliiseS — APPROVED** [review]

**Pull request merged**